src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:39157/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:39157/project/issues?id=sample&pullRequest=6797&issues=AWlIOSQ6MKt28h7PFB_R&open=AWlIOSQ6MKt28h7PFB_R)

[ISSUE_KEY]: AWlIOSQ6MKt28h7PFB_R